cluster/os-3.9.0-alpha.4/provider.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -e
+
+prefix=kubevirt-os-3.9.0-alpha.4
+
+function _main_ip() {
+    echo 127.0.0.1
+}
+
+_cli='docker run --privileged --rm -v /var/run/docker.sock:/var/run/docker.sock rmohr/cli:latest'
+
+function up() {
+    # Add one, 0 here means no node at all, but in the kubevirt repo it means master-only
+    local num_nodes=${VAGRANT_NUM_NODES-0}
+    num_nodes=$((num_nodes + 1))
```

Comment:
in the README it states that only a single node works right now. It looks like there's logic here to bring up multiple nodes though.

Reply:
Yes, I thought maybe we will have support for the multi-node environment before this PR will be merged :smile: